src/mSupplyMobileApp.js

```diff
@@ -57,6 +56,7 @@ class MSupplyMobileAppContainer extends React.Component {
     this.userAuthenticator = new UserAuthenticator(this.database, this.settings);
     const syncAuthenticator = new SyncAuthenticator(this.settings);
     this.synchroniser = new Synchroniser(database, syncAuthenticator, this.settings);
+    this.postSyncProcessor = new PostSyncProcessor(this.database);
     this.scheduler = new Scheduler();
     const initialised = this.synchroniser.isInitialised();
```

Comment:
Can we change `initialised` to `isInitialised`? Been caught out on a boolean recently, good to have the indicator